examples/chef/.shell_env

```diff
@@ -0,0 +1,51 @@
+SHELL=/usr/bin/zsh
```

Comment:
this and other `chef/.*` files should not be checked in. I believe they should not be created anymore by latest chef builds.